.github/workflows/multios-unit-tests.yml

```diff
@@ -42,8 +42,7 @@ jobs:
         shell: bash
         run: |
             go list ./... | grep -v -e grpc.v12 -e google.golang.org/api -e sarama -e confluent-kafka-go -e cmemprof | sort >packages.txt
-            gotestsum --junitfile ${REPORT} -- $(cat packages.txt) -v -coverprofile=coverage.txt -covermode=atomic -timeout=15m
-
```

Comment:
Good catch, no we don't. In fact I've removed the timeout flag. This will bring us back to the default 10 minute timeout, which should now be okay. The last windows run I tried on this branch took 8 minutes, and we can likely make that even faster by doing something similar to this PR for other tests.

Reply:
`we can likely make that even faster by doing something similar to this PR for other tests.`
I can create a ticket to track that work